mypy/build.py

```diff
@@ -474,8 +473,9 @@ def __init__(self, data_dir: str,
         self.version_id = version_id
         self.modules = {}  # type: Dict[str, MypyFile]
         self.missing_modules = set()  # type: Set[str]
+        self.plugin = plugin
```

Comment:
@JukkaL you have `self.plugin = plugin` twice in the init (on 476 and 485 at the end of the init). Is that on purpose?